api/types.proto

```diff
@@ -558,7 +558,7 @@ message EncryptionKey {
 	uint64 lamport_time = 4;
 }
 
-// Manager provides informations about the state of a manager in the cluster.
-message Manager {
+// ManagerStatus provides information about the status of a manager in the cluster.
+message ManagerStatus {
 	RaftMember raft = 1 [(gogoproto.nullable) = false];
-}
+}
```

Comment:
nit: How come these final newlines keep disappearing from the proto files? It shouldn't break anything but it's a little weird for a text file not to end with a newline (and other people's editors will re-add it next time someone else edits the file).